webapp/src/views/CrudList.vue

```diff
@@ -204,20 +204,11 @@
               </v-layout>
             </template>
             <template slot="no-data">
-              <div
-                v-if="$store.state.working"
-                class="py-5 text-xs-center"
-              >
-                <p>fetching data ...</p>
-              </div>
-              <div
-                v-else
-                class="py-5 text-xs-center"
-              >
+              <div class="py-4 text-xs-center">
                 <h2 class="title">
                   Feels so empty here!
                 </h2>
-                <p>No data yet.</p>
+                <p>No entries yet.</p>
```

Comment:
This message is misleadingly displayed during loading

Reply:
I tested this many times, and though I had verified that it displays the standard v-datatable loading message ("fetching data" or something). Can you let me know how to reproduce the issue?

Reply:
Indeed, it looks fine. Not sure anymore when/where I saw that.